src/main/resources/content/cft/AWS-S3-Object-Versioning-is-disabled.json

```diff
@@ -1 +1,8 @@
-{"severity":"medium","rule":"$.Resources.[*].Type contains AWS::S3::Bucket and ($.Resources.[*].Properties.VersioningConfiguration does not exist or ($.Resources.[*].Properties.VersioningConfiguration exists and $.Resources.[*].Properties.VersioningConfiguration.Status does not equal Enabled))","id":"8ec3f878-0f5e-4782-b4cd-98018b217be5","enabled":true,"resourceType":"s3","policy":"AWS S3 Object Versioning is disabled"}
\ No newline at end of file
+{
+  "severity": "medium",
+  "rule": "$.Resources.[*].Type contains AWS::S3::Bucket and ($.Resources.[*].Properties.VersioningConfiguration does not exist or ($.Resources.[*].Properties.VersioningConfiguration exists and $.Resources.[*].Properties.VersioningConfiguration.Status contains Suspended))",
```

Comment:
The rule is not correct. The Resources.[*] will check for any resource having VersioningConfiguration. It will not be specific to AWS::S3::Bucket. Pls fix the rule.